package.json

```diff
@@ -53,7 +53,6 @@
     "start:docker:embed": "cd libs/embed && npm run start:docker",
     "start:docker:web": "cross-env nx run-many --target=start:docker --projects=@novu/web",
     "start:docker:widget": "cross-env nx run-many --target=start:docker --projects=@novu/widget",
-    "start:e2e:api": "cd apps/api && pnpm run test:e2e",
```

Comment:
now used across the project, removed it so it wont confuse people, test:e2e does not exist on `@novu/api`